CHANGELOG.md

```diff
@@ -5,3 +5,4 @@
 - Add `Ezcater/RspecRequireFeatureFlagMock` cop.
 - Add `Ezcater/RspecDotNotSelfDot` cop.
 - Add `Ezcater/StyleDig` cop.
+- Add `Ezcater/RspecRequireBrowserMock` cop.
```

Comment:
I released this version of the gem, so the addition should go into a `v0.49.1`.

Reply:
ok - sorry - didn't realize that.